_posts/randy/2022-07-03-all-old-movies-are-garbage.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "All Old Movies Are Garbage (In the End)"
+permalink: /@randy/old-movies-are-garbage
+image: "assets/img/old-movies-are-garbage.png"
+description: Mine 
+tags: tails analog-sampling
+---
+
+![Old Movies Are Garbage](/assets/img/old-movies-are-garbage.png)
```

Comment:
Do you want the image and the video at the top?  Last time we just had the image as the preview that shows when the link is shared and only the video on the actual article

Reply:
same as last time